reference/Upload-API.yaml

```diff
@@ -265,6 +224,143 @@ paths:
+                  message:
+                    type: string
+                    minLength: 1
+                required:
+                  - status
+                  - message
+              examples:
+                Example:
+                  value:
+                    status: 200
+                    message: Attributes updated successfully
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                description: ''
+                type: object
+                properties:
+                  status:
+                    type: number
+                    enum:
+                      - 400
+                  message:
+                    type: string
+                    minLength: 1
+                required:
+                  - status
+                  - message
+              examples:
+                Example:
+                  value:
+                    status: 400
+                    message: Attribute id e078d72a-aad4-4b68-9227-e0bde0b35c6d not found
+      description: |-
+        <!--
+        focus: false
+        -->
+        ![In Review](https://img.shields.io/badge/Status-In%20Review-green?style=flat-square)
+
+        The update attributes endpoint allows updating existing attributes in the CloudMatrix.
+      tags:
+        - Update
+      requestBody:
+        content:
+          application/json:
+            schema:
+              description: ''
+              type: object
+              properties:
+                attributes:
+                  type: array
+                  uniqueItems: true
+                  minItems: 1
+                  description: The metadata attributes
+                  items:
+                    type: object
+                    properties:
+                      id:
+                        type: string
+                        minLength: 1
+                        description: CloudMatrix attribute id
+                      label:
+                        type: string
+                        minLength: 1
+                        description: CloudMatrix attribute label
+                      value:
+                        type: string
+                        minLength: 1
+                        description: CloudMatrix attribute value
+                      items:
+                        type: array
+                        uniqueItems: true
+                        minItems: 1
+                        description: CloudMatrix attribute where value is a collection
+                        items:
+                          type: object
+                          additionalProperties: false
+                          properties:
+                            alias:
+                              type: string
+                              minLength: 1
+                              description: 'CloudMatrix attribute report value, this should be used as an alias for 3rd party attribute value '
```

Comment:
Noticed this description has quotes around it whilst others don't !?

Reply:
That's weird... 🤔  it's stoplight, not something done manually.
I'll remove it anyway 🤷